api/documentation/docs/tutorials/create-a-collective-offer.md

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_position: 3
+---
+
+# Create a collective offer
+
+## Setup
+
+First, setup a provider, if not already done (more information [**here**](/docs/mandatory-steps/request-a-provider-account#how-to-get-a-provider-account).
+
+Next step is to ensure that link your API key to at least one venue. Please note that inside the integration environment, all venues are considered known by Adage. This will not be the case in real life, please ensure that your linked venues can be used for collective offers.
+
+## API: create a bookable offer
+
+Here's an example.
+
+`POST /v2/collective/offers/`
+
+```js
+{
+  "venueId": 1234,
+  "name": "Atelier de peinture",
+  "description": "Describe your offer",
+
+  "subcategoryId": "ATELIER_PRATIQUE_ART",
+  "formats": [
+    "Atelier de pratique"
+  ],
+
+  "bookingEmails": [
+    "notifications.update@email.com"
+  ],
+  "contactEmail": "somebody.to.contact@gmail.com",
+  "contactPhone": "0908070605",
+  "domains": [
+    1,
+    2
+  ],
+  "students": [
+    "GENERAL2",
+    "GENERAL1",
+    "GENERAL0"
+  ],
+  "audioDisabilityCompliant": true,
+  "visualDisabilityCompliant": true
+  "mentalDisabilityCompliant": true,
+  "motorDisabilityCompliant": true,
+  "offerVenue": {
+    "addressType": "offererVenue",
+    "otherAddress": null,
+    "venueId": 1234
+  },
+  "isActive": true,
+
+  "beginningDatetime": "2024-07-25T14:00:00+02:00",
+  "startDatetime": "2024-07-25T14:00:00+02:00",
+  "endDatetime": "2024-07-25T19:00:00+02:00",
+
+  "bookingLimitDatetime": "2024-07-25T14:00:00+02:00",
+  "totalPrice": 100,
+  "numberOfTickets": 10,
+
+  "educationalInstitution": "0010008D",
+  "nationalProgramId": 2,
+
+
+
+  "durationMinutes": 60,
+  "imageFile": "",
+  "imageCredit": "",
+  "educationalPriceDetail": "10 tickets x 10 € = 100 €",
+
+}
+```
```

Comment:
```suggestion
Your JSON payload should look something like :

```json
{
  "venueId": 1234,
  "name": "Atelier de peinture",
  "description": "Describe your offer",
  "subcategoryId": "ATELIER_PRATIQUE_ART",
  "formats": [
    "Atelier de pratique"
  ],
  "bookingEmails": [
    "notifications.update@email.com"
  ],
  "contactEmail": "somebody.to.contact@gmail.com",
  "contactPhone": "0908070605",
  "domains": [
    1,
    2
  ],
  "students": [
    "GENERAL2",
    "GENERAL1",
    "GENERAL0"
  ],
  "audioDisabilityCompliant": true,
  "visualDisabilityCompliant": true
  "mentalDisabilityCompliant": true,
  "motorDisabilityCompliant": true,
  "offerVenue": {
    "addressType": "offererVenue",
    "otherAddress": null,
    "venueId": 1234
  },
  "isActive": true,
  "beginningDatetime": "2024-07-25T14:00:00+02:00",
  "startDatetime": "2024-07-25T14:00:00+02:00",
  "endDatetime": "2024-07-25T19:00:00+02:00",
  "bookingLimitDatetime": "2024-07-25T14:00:00+02:00",
  "totalPrice": 100,
  "numberOfTickets": 10,
  "educationalInstitution": "0010008D",
  "nationalProgramId": 2,
  "durationMinutes": 60,
  "imageFile": "",
  "imageCredit": "",
  "educationalPriceDetail": "10 tickets x 10 € = 100 €"
}
```
```